base/all-all/sbardef.lmp

```diff
@@ -90,6 +90,100 @@
                 {
                   "condition": 4,
                   "param": 0
+                },
+                {
+                  "condition": 22,
+                  "param": 0
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "number":
+            {
+              "x": 44,
+              "y": 3,
+              "alignment": 2,
+              "font": "BigRed",
+              "tranmap": null,
+              "translation": "CRRED",
+              "type": 4,
+              "param": 0,
+              "maxlength": 3,
+              "conditions":
+              [
+                {
+                  "condition": 4,
+                  "param": 0
+                },
+                {
+                  "condition": 20,
```

Comment:
What I meant with "namespace": Is it possible to e.g. introduce a "condition-woof" field for conditions >= 20, that other ports simply don't parse? Or will they error out when they encounter an unknown field name?

Reply:
We can make a large container (“canvas”) with a “boom” condition, or simply define `statusbar[3]`, which is ignored in the current specification.

Alternatively, we can increase the lump version so that the KEX port ignores our SBARDEF.